.size-limit.js

```diff
@@ -28,7 +28,7 @@ module.exports = [
     name: 'The size of the @material-ui/core modules',
     webpack: true,
     path: 'packages/material-ui/build/index.js',
-    limit: '91.2 KB',
+    limit: '94.7 KB',
```

Comment:
I think this is not branched of the latest `next` commit. Looks like this is missing #13391

Reply:
I rebased on my last push but didn't update the size-limit, thanks for the reminder